shared/macros/10-ansible.jinja

```diff
@@ -771,6 +771,37 @@ The following macro remediates Audit syscall rule in :code:`/etc/audit/audit.rul
 {{%- endmacro -%}}
 
 
+{{#
+Ansible ansible.builtin.package wrapping macro, ensure package(s) are installed
+
+:param package: Package(s) to  ensure are installed
+:type package: str | list[str]
+:param name: Name used in task name. If not set, package names joined with comma and "and".
+:type name: None | str
+
+#}}
+{{%- macro ansible_package_install(package, name=none) -%}}
+{{%- if (package is sequence) and (package is not mapping) and (package is not string) -%}}
+{{%- set packages = package -%}}
+{{%- else %}}
+{{%- set packages = [package] -%}}
+{{%- endif %}}
+{{%- if name is none %}}
+{{%- set name = "the " ~ (packages | join_comma_and) -%}}
+{{%- endif %}}
+- name: "{{{ rule_title }}} - Ensure {{{ name }}} {{% if packages | length > 1 %}}Packages Are{{% else %}}Package Is{{% endif %}} Installed"
```

Comment:
It would be better if the overridden package name would be also used in the task name.